<!DOCTYPE html>
{% load static %}
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Omic Horizon Expression</title>
	<link rel="stylesheet" type="text/css" href="{% static 'css/immusubset.css' %}">
	<link rel="stylesheet" type="text/css" href="{% static 'css/FAQ.css' %}">
    <!-- Bootstrap Core CSS -->
    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet" type="text/css">

	<!-- Font Awesome -->
	<link href="{% static 'font-awesome/css/font-awesome.min.css' %}" rel="stylesheet" type="text/css">

    <!-- Animation -->
	<link href="{% static 'css/animate.css' %}" rel="stylesheet" />

    <!-- MyTemplate CSS -->
    <link href="{% static 'css/style.css' %}" rel="stylesheet">
	<link href="{% static 'css/contact.css' %}" rel="stylesheet">

	<!-- multiple-select CSS	-->
	<link rel="stylesheet" href="{% static 'css/multiple-select.min.css' %}">

	<!-- Core JavaScript Files -->
	<script src="{%static 'js/jquery.min.js' %}" type="text/javascript"></script>
	<script src="{%static 'js/bootstrap.min.js' %}" type="text/javascript"></script>
	<script src="{%static 'js/jquery.easing.min.js' %}" type="text/javascript"></script>
	<script src="{%static 'js/jquery.scrollTo.js' %}" type="text/javascript"></script>
	<script src="{%static 'js/wow.min.js' %}" type="text/javascript"></script>			<!-- Reveal animation when you scroll by wow.js. It need animate.css library -->
	<!-- Custom Theme JavaScript -->
	<script src="{%static 'js/custom.js' %}" type="text/javascript"></script>
	<script src="{%static 'js/echarts.js' %}" charset="utf-8"></script>

	<!-- normal function JavaScript	-->
	<script src="{%static 'js/normal.js' %}" type="text/javascript"></script>

	<!-- multiple-select JavaScript	-->
	<script src="{%static 'js/multiple-select.min.js' %}" type="text/javascript"></script>
</head>

<body>

	<header id="header-banner">
		<nav class="navbar navbar-default navbar-fixed-top fadeIn" role="navigation">
			<div class="container">
				<!-- Brand and toggle get grouped for better mobile display -->
				<div class="navbar-header">
					<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#dropdown-box-1">
						<span class="sr-only">Toggle navigation</span>
					</button>
					<div class="navbar-brand">
						<a href="#" style="font-size:16px;">FAQ</a>
					</div>
				</div>

				<!-- Collect the nav links and other content for toggling -->
				<div class="collapse navbar-collapse navbar-right" id="dropdown-box-1">
					<ul class="nav navbar-nav">
						<li><a href="/OmicHorizonExpression/search/">Home</a></li>
						<li><a href="/OmicHorizonExpression/update/">Update</a></li>
						<li><a href="/OmicHorizonExpression/contact/">Contact</a></li>
						<li class="active"><a href="/OmicHorizonExpression/FAQ/">FAQ</a></li>
						<li><a href="http://immudb.bjmu.edu.cn/" target="_blank">ImmuDB</a></li>
					</ul>
				</div>
			</div> <!-- /.container -->
		</nav> <!-- /.nav -->
	</header>

	<!-- aboutus -->
    <section class="aboutus" id="aboutus">
		<div class="container">
			<div class="row">
				<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
					<div id="bd">
						<div id="content" style="width:100%;margin-top:5%;" >
                            <h2>About this database</h2>
                            <h3 style="margin-left:2em;">
                                <a href="#first">1. What does the Omic Horizon Expression database do?</a>
                            </h3>
                            <h3 style="margin-left:2em;">
                                <a href="#second">2. Where are the data come from?</a>
                            </h3>
							<h3 style="margin-left:2em;">
                                <a href="#third">3. How can the database be queried?</a>
                            </h3>
							<h3 style="margin-left:2em;">
                                <a href="#fourth">4. How to understand the query result?</a>
                            </h3>
							<h3 style="margin-left:2em;">
                                <a href="#fifth">5. What is a TPM value?</a>
                            </h3>
						</div>
    				</div>
				</div>
			</div>
		</div>
	</section>
	<!-- /.aboutus -->

	<section class="services" id="services">
		<div class="container">
			<div class="row-padded" id="first" style="padding:10px;"></div>
			<div class="row">
				<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
					<div class="col-sm-7" style="width:100%;">
						<h4>1. What does the Omic Horizon Expression database do?</h4>
						<p style="font-size:16px;">
							Omic Horizon Expression (or OmicHorizon@Expression) is a database of gene expression in multiple tissues, primary cells and cell lines based on omics data. It aims to show mRNA expression levels by TPM values from bulk RNA sequencing (RNA-Seq) data for multiple species. However, only rat samples are included in the current version. These samples are divided into three categories, including tissues, primary cells and cell lines. Some tissues are further divided into several different subtissues according to the anatomical site. As an omic resource, the database provides horizons of gene expression across various conditions.
						</p>
						<a href="#aboutus" class="Back">Back to Index</a>
					</div>
				</div>
			</div>
			<hr style="border-color:#d2d2d2;">
			<div class="row-padded" id="second" style="padding:0px;"></div>
			<div class="row">
				<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
					<div class="col-sm-7" style="width:100%;">
						<h4>2. Where are the data come from?</h4>
						<p style="font-size:16px;">
							We collected rat RNA-Seq data from the Sequence Read Archive (SRA) database. The downloaded sra format files were uniformly processed, including file format conversion, quality control, reads mapping (reference genome: mRatBN7.2) and gene expression quantification by TPM values. The current version contains over 2,600 samples, which are categorized into 48 tissues, 40 primary cell types, and 25 cell lines.
						</p>
						<a href="#aboutus" class="Back">Back to Index</a>
					</div>
				</div>
			</div>
			<hr style="border-color:#d2d2d2;">
			<div class="row-padded" id="third" style="padding:0px;"></div>
			<div class="row">
				<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
					<div class="col-sm-7" style="width:100%;">
						<h4>3. How can the database be queried?</h4>
						<p style="font-size:16px;">
							The OmicHorizon@Expression database supports requests using gene symbols (or alias), Ensemble and Entrez gene IDs. The interface is simple and easy to understand. A basic operation and work flow are shown as follows:<br>
							(1) Select a search type, such as tissue, primary cell and cell line, from the drop-down menu. Then, an interesting tissue or cell type is selected before the new drop-down menu pops up for further selection of subtissue or cell sources. When ‘All’ is selected, all tissues or cells under the layer will be queried to show the result.<br>
							(2) Enter your query gene symbol or gene ID, and click on the button ‘GO’ to enter the result page for the gene expression profile.<br>
							(3) Please note that not all legends on the X-axis will be displayed when there are too many retrieved tissues or sub-tissues, for example, when "All" is selected in the first step.<br>
						</p>
						<a href="#aboutus" class="Back">Back to Index</a>
					</div>
				</div>
			</div>
			<hr style="border-color:#d2d2d2;">
			<div class="row-padded" id="fourth" style="padding:0px;"></div>
			<div class="row">
				<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
					<div class="col-sm-7" style="width:100%;">
						<h4>4. How to understand the query result?</h4>
						<p style="font-size:16px;">
							The result page includes three sections:<br>
							Section 1. Basic information of the queried gene, such as gene description, gene alias, gene IDs, gene biotype and chromosome location.<br>
							Section 2. Gene expression profile shown in a box plot. In the plot, the x-axis represents the queried tissues, primary cells or cell lines, while the y-axis represents the expressional TPM values (see below). A box plot consists of six data nodes, which arrange a set of data from largest to smallest, and calculate its upper edge (Q3+1.5*IQR), Q3 (75th percentile), median (50th percentile), Q1 (25th percentile), lower edge (Q1-1.5*IQR) and abnormal values in circle in the plot. The upper and lower edges are set to the maximum and minimum values, respectively, when there are no abnormal values. Several prompt messages by mouse are shown in the following figure.<br>
							Section 3. A table form shows the gene expression values of the queried gene in the interested tissues or cells. Each value represents a sample source.<br>
							The box plot and table files are downloadable and can be saved to a hard drive.<br>
						</p>
						<p style="text-align:center;">
							<img src="/static/img/FAQ4img.jpg" width="1000px;">
						</p>
						<a href="#aboutus" class="Back">Back to Index</a>
					</div>
				</div>
			</div>
			<hr style="border-color:#d2d2d2;">
			<div class="row-padded" id="fifth" style="padding:0px;"></div>
			<div class="row">
				<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
					<div class="col-sm-7" style="width:100%;">
						<h4>5. What is a TPM value?</h4>
						<P style="font-size:16px;">
							To eliminate the effects of sequencing depth and gene length, we used the TPM (transcripts per kilobase million) value to measure the gene expression level. A TPM value is calculated by the following formula:<br>
							TPM=(Ni/Li)*1,000,000/sum(Ni/Li+…….+ Nm/Lm)<br>
							Ni: the count of reads mapping to gene i<br>
							Li: the length of exons of gene i<br>
						</P>
						<a href="#aboutus" class="Back">Back to Index</a>
					</div>
				</div>
			</div>
		</div>
	</section>

	<!-- footer -->
	<footer id="section-footer">
		<div class="container">
			<div class="row">
				<div class="col-md-12 col-lg-12">
					<div class="wow fadeIn" data-wow-delay="0.4s">
						<div class="btn-circle-scroll">
							<a href="#header-banner" class="btn-circle">
								<i class="fa fa-angle-double-up animated"></i>
							</a>
						</div>
					</div>
					<p>All Rights Reserved. Department of Immunology, Peking University Health Science Center.</p>
					<p>Designed by Department of Natural Science in Medicine, Peking University Health Science Center.</p>
					<p>Supported by Information and Communication Center, Peking University Health Science Center</p>
				</div>
			</div>
		</div>
	</footer>
	<!-- /.footer -->
</body>

</html>